identity/identity.proto

```diff
@@ -200,6 +200,27 @@ message Capability {
     Type type = 1;
   }
 
+  // EncryptionKeyRotation contains the features of the EncryptionKeyRotation
+  // operation that the CSI driver supports.
+  message EncryptionKeyRotation {
+    // Type describes a CSI Encryption Service that CSI driver can support.
+    enum Type {
+      // UNKNOWN indicates that the CSI driver does not support the
+      // EncryptionKeyRotation operation in the current mode.
+      // The CSI-Addons CO plugin will most likely ignore this node
+      // for the EncryptionKeyRotation operation.
+      UNKNOWN = 0;
+      // ENCRYPTIONKEYROTATION indicates that the CSI driver provides
+      // RPCs for an EncryptionKeyRotation operation.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require a volume to be staged/attached to
```

Comment:
can you please check on this one. what do you mean by `invole RPCs that require a volume to be staged/attached to the node` isn't it already staged/attached?

Reply:
Ah, an overlook on my part. Changing it to:

```
// The presence of this capability determines whether the CSI-Addons CO
// plugin can invoke RPCs for rotating the encryption key.
```